### PR TITLE
Fix issues on displaying tenant scope information in consent screen

### PR DESCRIPTION
**Pull request by ChanikaRuchini**
### Purpose
$subject

### Related Issue
https://github.com/wso2/product-is/issues/11198

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation.
- [ ] Documentation provided. (Add links if there's any)
- [ ] Unit tests provided. (Add links if there's any)
- [ ] Integration tests provided. (Add links if there's any)

### Related PRs
Merge after [identity-inbound-auth-oauth#671 ](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1671)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/authentication-portal/src/main/webapp/oauth2_consent.jsp [hunk 1]
```diff
@@ -42,6 +46,18 @@
     String scopeString = request.getParameter("scope");
     boolean displayScopes = Boolean.parseBoolean(getServletContext().getInitParameter("displayScopes"));
 
+    Map<String, String> queryParamMap = new HashMap<String, String>();
+            String queryString = request.getParameter("spQueryParams");
```
Comment (brionmario) [inline apps/authentication-portal/src/main/webapp/oauth2_consent.jsp:50]: Lets fix the formatting issue here.

**Review by thivi — APPROVED** [review]

File: apps/authentication-portal/src/main/webapp/oauth2_consent.jsp (hunk 1 above)
Comment (ChanikaRuchini) [reply apps/authentication-portal/src/main/webapp/oauth2_consent.jsp]: fixed

File: apps/authentication-portal/src/main/webapp/oauth2_consent.jsp [hunk 2]
```diff
@@ -42,6 +46,18 @@
     String scopeString = request.getParameter("scope");
     boolean displayScopes = Boolean.parseBoolean(getServletContext().getInitParameter("displayScopes"));
 
+    Map<String, String> queryParamMap = new HashMap<String, String>();
+    String queryString = request.getParameter("spQueryParams");
+        if (StringUtils.isNotBlank(queryString)) {
+            StringTokenizer stringTokenizer = new StringTokenizer(queryString, "&");
```
Comment (ashensw) [inline apps/authentication-portal/src/main/webapp/oauth2_consent.jsp:52]: Can't we use `String.split()` here?

Comment (ChanikaRuchini) [reply apps/authentication-portal/src/main/webapp/oauth2_consent.jsp]: Fixed

**Comment by ashensw** [comment]
@ChanikaRuchini bump the OAuth version before merging this.

**Review by ashensw — APPROVED** [review]

**Pull request merged**